src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -119,4 +123,19 @@ public void moveArmToPosition() {
         .getPIDController()
         .setReference(armPositions.get(currentArmPosition), CANSparkMax.ControlType.kPosition);
   }
+
+  //controller
+  public final CommandXboxController m_driverController = new CommandXboxController(0);
+  
+  //buttons
+  Trigger aTrigger = m_driverController.a();
+  Trigger bTrigger = m_driverController.b();
+  Trigger xTrigger = m_driverController.x();
+  Trigger yTrigger = m_driverController.y();
+
+  public EventLoop getDefaultButtonLoop(){
+    //I need to use methods like 'a()' with type 'Trigger' inside this DefualtButtonLoop in order to use the commands like runToScore
+    //But i'm not sure how
+    return null;
+ }
```

Comment:
Don't worry about the controller in this file- all controller inputs would be handled by RobotContainer.java or by a command using the subsystem. We generally want to keep subsystems as simple as possible, so this can be removed.